packages/design-system/src/components/Form/Primitives/Primitives.stories.tsx

```diff
@@ -125,3 +184,34 @@ export const InputPrimitiveWithDropdown = () => {
 		</StackVertical>
 	);
 };
+
+export const FakeForm = () => {
```

Comment:
I would love to have a story using controlled input with RHF and unit test, but let's wait for SB 6.5 and it's test runner

Reply:
It's in my plans down the line when I've built the actual `Field` components!